fastai/callbacks/hooks.py

```diff
@@ -181,6 +184,18 @@ def model_summary(m:Learner, n:int=70):
     res += f"\nTotal params: {total_params:,}\n"
     res += f"Total trainable params: {total_trainable_params:,}\n"
     res += f"Total non-trainable params: {total_params - total_trainable_params:,}\n"
+           
+    res += f"Optimized with {str(m.opt_func)[25:-1].replace('>', '')}\n"
+    if m.true_wd and "adam" in str(m.opt_func).lower():
+        res += f"Using true weight decay as discussed in https://www.fast.ai/2018/07/02/adam-weight-decay/ \n"
+    res += f"Loss function : {m.loss_func.__class__.__name__}\n"
+    res += "=" * n + "\n"
+    res += "Callbacks functions applied \n"
+    for cbk in m.callbacks:
+        if cbk.__doc__ is None:
```

Comment:
I'd remove the calls to __doc__ and only print the names: there is a `doc` function that fastai provides that will show the doc if needed, with the links to the source.